packages/compass-import-export/src/components/export-code-view.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useMemo } from 'react';
+import { connect } from 'react-redux';
+import { Body, Code, css, spacing } from '@mongodb-js/compass-components';
+
+import type { FieldsToExportOption, FieldsToExport } from '../modules/export';
+import type { RootExportState } from '../stores/export-store';
+import { createProjectionFromSchemaFields } from '../export/gather-fields';
+import type { ExportQuery } from '../export/export-types';
+import { newGetQueryAsShellJSString } from '../utils/get-shell-js';
+
+const containerStyles = css({
+  marginBottom: spacing[3],
+});
+
+type ExportCodeViewProps = {
+  ns: string;
+  query?: ExportQuery;
+  fields: FieldsToExport;
+  selectedFieldOption: undefined | FieldsToExportOption;
+};
+
+function ExportCodeView({
+  ns,
+  query,
+  fields,
+  selectedFieldOption,
+}: ExportCodeViewProps) {
+  const code = useMemo(() => {
+    if (selectedFieldOption === 'select-fields') {
+      return newGetQueryAsShellJSString({
+        query: {
+          ...(query ?? {
+            filter: {},
+          }),
+          projection: createProjectionFromSchemaFields(
```

Comment:
We should set `_id: false` when it's not selected to be sure it's not automatically included.

Reply:
done